src/test/java/io/confluent/connect/elasticsearch/integration/ElasticsearchConnectorBaseIT.java

```diff
@@ -146,6 +154,17 @@ protected void runSimpleTest(Map<String, String> props) throws Exception {
     verifySearchResults(NUM_RECORDS);
   }
 
+  protected void assertConnectorFails(Map<String, String> props, String trace) throws Exception {
```

Comment:
should we rename to assertConnectorFailsOnWriteRecords?

Reply:
Sure, renamed.